website/blog/2023-05-04-generating-dynamic-docs.md

```diff
@@ -0,0 +1,368 @@
+---
+title: "Generating dynamic dbt docs for a whole domain at once"
```

Comment:
fair point! I'm struggling a bit with a good title, and very open to suggestions :) Here's another suggestion:
```suggestion
title: "Accelerate your documentation workflow: Generate docs for whole folders at once"
```